SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -71,6 +71,12 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 	FReceptionistConfig ReceptionistConfig;
 	FLocatorConfig LocatorConfig;
 
+	DECLARE_DELEGATE(OnConnectionToSpatialOSSucceededDelegate)
+	OnConnectionToSpatialOSSucceededDelegate OnConnectedCallback;
```

Comment:
comparing with master and don't see this line there. is this part needed?

oh or is this from the ConnectionManager and we don't have that one in this rc?

Reply:
Yes I this was moved to the spatial connection manager. Fun fun fun.

Reply:
It exists in the `USpatialConnectionManager` :D